react-app/src/Home/Greeting.js

```diff
@@ -3,26 +3,26 @@ import React from 'react'
 import ThemeColor from '../Utilities/ThemeColor'
 
 export default function Greeting() {
-  const Greetings = () => {
-    const userDate = new Date()
-    const hours = userDate.getHours()
-    let greet
-    if (hours < 12) greet = 'Morning'
-    else if (hours >= 12 && hours <= 17) greet = 'Afternoon'
-    else if (hours >= 17 && hours <= 24) greet = 'Evening'
-    return <Text style={styles.greetingText}>Good {greet},</Text>
-  }
+  const userDate = new Date()
```

Comment:
Nice!

Reply:
Eric did that not me, I just modified it a bit to more simple.

Reply:
yep, i like the way you keep the code clean. its very good habit! like it